// 3

import "err"

let a = 3
err.println(a)

let b = a
